test/metrics_test.rb

```diff
@@ -130,13 +128,9 @@ def create_known_metrics(time = fixed_time)
 
       q = Sidekiq::Metrics::Query.new(now: fixed_time)
       result = q.top_jobs(class_filter: /some/i)
-      assert_equal fixed_time - 59 * 60, result.starts_at
+      assert_equal fixed_time - 60 * 60, result.starts_at
```

Comment:
To be honest I'm not sure what caused this to change, but it's the difference of one minute, so... 🤷‍♂️